config/config.go

```diff
@@ -1,4 +0,0 @@
-// Package config is responsible for loading user-manager application config.
-// Basic configuration like consul credentials and address, http port to listen for requests,
-// postgres schema name, credentials, and client timeout are read from environment variables.
```

Comment:
Code from the previous contributor removed
Why?

Reply:
Because this code was merge by mistake to this branch

Reply:
As soon as you merge PR, 93 lines of code from @i3odja will be deleted
Is it okay?

Reply:
fix it

Reply:
`go.mod` marked as removed

Reply:
fix it